apps/docs/src/.vitepress/components/ComponentRoadmap.vue

```diff
@@ -55,7 +55,7 @@ const shouldShowAllButton = computed(() => {
   <section class="components">
```

Comment:
I noticed something weird. the "Components" headline looks different in storybook vs the vitepress docs on your branch.

![image](https://github.com/SchwarzIT/onyx/assets/151019977/4adbd118-4670-4d84-92e7-052a4113a874)

Reply:
Looks the same for me, did you run `pnpm install`?

Reply:
nope, still different.
![image](https://github.com/SchwarzIT/onyx/assets/151019977/f5fde108-eb12-4f68-96a5-58dd290dc05b)

Reply:
computed styles of the headline (h2) from vitepress
```
box-sizing border-box
color rgb(62, 89, 110)
display block
font-family "Source Sans 3 Variable", sans-serif
font-size 40px
font-synthesis-small-caps none
font-synthesis-style auto
font-synthesis-weight none
font-weight 600
height 40px
line-height 40px
margin-block-end 0px
margin-block-start 0px
margin-bottom 0px
margin-inline-end 0px
margin-inline-start 0px
margin-left 0px
margin-right 0px
margin-top 0px
text-rendering optimizelegibility
text-size-adjust 100%
unicode-bidi isolate
width 613px
-webkit-font-smoothing antialiased
```

from storybook 
```
box-sizing border-box
color rgb(62, 89, 110)
color-scheme light
display block
font-family "Source Sans 3 Variable", sans-serif
font-size 40px
font-synthesis-small-caps none
font-synthesis-style auto
font-synthesis-weight none
font-weight 400
height 40px
line-height 40px
margin-block-end 0px
margin-block-start 0px
margin-bottom 0px
margin-inline-end 0px
margin-inline-start 0px
margin-left 0px
margin-right 0px
margin-top 0px
overflow-wrap break-word
text-rendering optimizelegibility
text-size-adjust 100%
unicode-bidi isolate
width 291px
-webkit-font-smoothing antialiased
```

Reply:
font-weight differs.
left vitepress, right storybook

![image](https://github.com/SchwarzIT/onyx/assets/151019977/d516aa94-4eca-49ff-924f-2048fac5ac60)

Reply:
FYI: as discussed I will close this issue. The problem is that the "vp-raw" class does only work in VitePress, not Storybook